### PR TITLE
Add generating pdf files

### PR DESCRIPTION
**Pull request by apokralipsa**

### CODE REVIEW

File: vbaSrc/Employee [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Public organisationalUnit As String
+Public supervisorName As String
+Public supervisorSurname As String
+Public contractStartDate As String
+
+Sub gatherInfo()
+On Error GoTo cancel
+    name = getInput("Podaj imi�")
+    surname = getInput("Podaj nazwisko")
+    organisationalUnit = getInput("Podaj dzia� (np. YIAID)")
```
Comment (pawelpluta) [inline vbaSrc/Employee:23]: Can you try to fix encoding?

Comment (apokralipsa) [reply vbaSrc/Employee]: The vbe expects this (Windows) encoding. All file related operations (input, output, exporting modules etc.) use it.

Comment (apokralipsa) [reply vbaSrc/Employee]: In short - even if it is possible, it would be a massive hack.

Comment (apokralipsa) [reply vbaSrc/Employee]: As we talked offline - switched to english to avoid issues with encoding alltogether.

File: vbaSrc/Employee [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Public supervisorSurname As String
+Public contractStartDate As String
+
+Sub gatherInfo()
+On Error GoTo cancel
+    name = user.getInput("Input your name")
+    surname = user.getInput("Input your surname")
+    organisationalUnit = user.getInput("Input your organisational unit (e.g. YIAID)")
+    supervisorName = user.getInput("Input the name of your supervisor")
+    supervisorSurname = user.getInput("Input the name of your supervisor")
```
Comment (pawelnowakowski) [inline vbaSrc/Employee:25]: `name` => `surname`

Comment (apokralipsa) [reply vbaSrc/Employee]: Good point. Fixed.

**Review by pawelpluta — APPROVED** [review]

**Pull request merged**